course-mw/sunbird-util/sunbird-platform-core/common-util/src/main/java/org/sunbird/common/request/BaseRequestValidator.java

```diff
@@ -499,4 +501,12 @@ public void validateRequestedBy(String requestedBy) {
               ResponseCode.UNAUTHORIZED.getResponseCode());
     }
   }
+  public void validateParamCollection(List<String> value, ResponseCode error,String errorMsgArgument) {
+    if (CollectionUtils.isEmpty (value)||null==(value)) {
```

Comment:
CollectionUtils.isEmpty -- will check the given object is null or not... no need of additional null value... 
But you need to loop through the collection and make sure empty string is given as input

Reply:
1. removed this part in the code   ||null==(value) 
2. collection is iterated and throwing error for empty string,
 in CourseEnrollmentRequestValidator line 101 to 107
 
    **if (userIdList != null&& userIdList.size()<30) {
      for (String userId : userIdList) {
        validateParam(
                userId,ResponseCode.invalidParameterValue,
                JsonKey.USER_ID);**